### PR TITLE
Gremlins

### PR DESCRIPTION
**Pull request by ghost**
gremlin

Despite their looks, gremlins have the potential to become the most destructive force that you will ever encounter on this space station. Unlike other monsters, which just eat you, gremlins will set up the singularity engine and then release it, flood the station with N2O, or lock you in a room and shock the door.

Gremlins have 20 hp so they die very quickly from any sort of combat activities

Gremlins possess the ability to manipulate some machinery and computers. You'll most commonly see them screwing up APCs, air alarms and airlocks, but they can do much more - for example us manipulate power and atmospherics equipment...

They can spawn in one of MANY locations during the Gremlins event (weight of 20. for comparison: grid check has a weight of 10, appendicitis has a weight of 20)
:cl: Thefastfoodguy
add: Gremlins
/:cl:

### CODE REVIEW

File: code/datums/wires/wires.dm [hunk 1]
```diff
@@ -74,7 +74,7 @@
 	"white",
 	"yellow"
 	)
-
```
Comment (j-mie) [inline code/datums/wires/wires.dm]: don't edit this line

**Comment by DinduStuffin** [comment]
>gremlins will set up the singularity engine and then release it, flood the station with N2O, or lock you in a room and shock the door.

>They can spawn in one of MANY locations during the Gremlins event (weight of 50. for comparison: grid check has a weight of 10, appendicitis has a weight of 20)

please no
it'll be like the gorillapocalypse all over again

I love the idea, don't get me wrong, but tone down the weight a bit so it's rare, or make it so that the location is announced. Announcing it would make the crew wanna go see em and/or fight em, and be aware of their location.

**Comment by PopNotes** [comment]
+1 once weight gets lowered
grem

**Comment by ghost** [comment]
reduce the weights significantly please
i'd much prefer they were too rare rather than too common

**Comment by ghost** [comment]
Weight lowered

**Comment by tortellinitony** [comment]
Laying and dead sprites?

**Comment by ghost** [comment]
yeah they're kinda the same... wait can gremlins lay down

**Comment by PopNotes** [comment]
I now approve of gremblins

**Review by GuyonBroadway — APPROVED** [review]
This looks ok, I'd like to see a testmerge before we hit the go button.

**Comment by ghost** [comment]
It's ok to merge, everything works.

**Comment by ghost** [comment]
garmfielf, you bad greblim, where is the lasaga
i gotta have a good meal.....

+1

**Comment by doobiedoo23** [comment]
these sound ridiculously annoying

+1

**Comment by ghost** [comment]
Destroy All Gremlins
+1

**Pull request merged**